main/tests/server/src/com/google/refine/commands/CommandTestBase.java

```diff
@@ -47,9 +50,11 @@ protected void assertCSRFCheckFailed() {
     /**
      * Convenience method to check that CSRF protection was NOT triggered
      */
-    protected void assertErrorNotCSRF() {
+    protected void assertErrorNotCSRF() throws JsonProcessingException {
         String response = writer.toString();
-        assertTrue(response.contains("\"code\":\"error\""));
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode node = mapper.readValue(response, JsonNode.class);
```

Comment:
Thank you! `ObjectMapper` objects are sort of expensive to instantiate so Jackson recommends to reuse a centrally-initialized instance. You could just do that by using `ParsingUtilities.mapper`.